RFC-004-Bitcoin.md

```diff
@@ -0,0 +1,79 @@
+# Bitcoin Definitions
+
+- RFC-Number: 004
+- Status: Draft
+- Discussion-issue: -
+- Created on: 26 Feb. 2019
+
+# Table of contents
+
+- [Description](#description)
+- [Motivation](#motivation)
+- [Content](#content)
+- [Registry extension](#registry-extension)
+
+## Description
+
+Introduces definitions to allow the execution of RFC-003 for Bitcoin asset on the [Bitcoin Core](https://github.com/bitcoin/bitcoin/) network.
+
+TODO: Decide we if do 1 RFC that introduce general Bitcoin terms + RFC003 or 2 RFCs (one for each)
+
+This includes the exact Bitcoin scripts to be used.
+
+TODO: Do we want p2sh(p2wsh) support? (more wallets support it)
```

Comment:
decision: decide on P2SH when finalising the document. Include P2SH open question to the tracking issue.